### PR TITLE
Updated markup and text improvements

### PR DESCRIPTION
**Pull request by Migaroez**
- Pulled the markup inline with the previous tutorial page
- Added a tip for naming pages
- Added some punctuation and removed leading text to make some paragraphs easier to read.

### CODE REVIEW

File: Tutorials/Creating-Basic-Site/Creating-Your-First-Template-and-Content-Node/index.md [hunk 1]
```diff
@@ -4,60 +4,68 @@ versionFrom: 8.0.0
...
+:::tip
+This Name will show up in the node list and will be used to create a url for the page. Try to keep it short but descriptive.
+:::
 
 ![Create a Homepage](images/figure-15a-create-a-homepage-enter-name-v8.png)
 
-Fill in the following on the **_Contents_** tab:
+Fill in the following **_properties_** underneath "Contents":
+
+```Page Title```
```
Comment (sofietoft) [inline Tutorials/Creating-Basic-Site/Creating-Your-First-Template-and-Content-Node/index.md]: I think we should keep these on the same line. That makes it clearer exactly what you need to change, right? 😁

File: Tutorials/Creating-Basic-Site/Creating-Your-First-Template-and-Content-Node/index.md [hunk 2]
```diff
@@ -4,60 +4,68 @@ versionFrom: 8.0.0
 
 # Creating (Editing) Your First Template
 
-Next click the expand node icon (it's the small triangle) in front of the **_Settings >_** **_Templates folder_** - you should then see a child node titled "_Homepage_"
+Next click the expand node icon (it's the small triangle) in front of the [Settings] > [Templates folder] - you should then see a child **_node_** titled "Homepage"
 
-Umbraco created this automatically when we created the **_Document Type_** (remember that option that said "..without template", we chose the one with and this is what it did).
+Umbraco created this automatically when we created the **_document type_** (remember that option that said "..without template", we chose the one with and this is what it did).
```
Comment (sofietoft) [inline Tutorials/Creating-Basic-Site/Creating-Your-First-Template-and-Content-Node/index.md]: A couple of times you've changed "Document Type" to "document type" .. as it's a concept within Umbraco, we prefer writing it with capital letters 😄

```suggestion
Umbraco created this automatically when we created the **_Document Type_** (remember that option that said "..without template", we chose the one with and this is what it did).
```

File: Tutorials/Creating-Basic-Site/Creating-Your-First-Template-and-Content-Node/index.md (hunk 1 above)
Comment (Migaroez) [reply Tutorials/Creating-Basic-Site/Creating-Your-First-Template-and-Content-Node/index.md]: I thought so to, but then you have a problem when a property value needs one than more line, like in an RTE value. I tried multiple layouts and this seemed the most concise. I am open to suggestions tough, or if you decide it should always be on one line, sure.

File: Tutorials/Creating-Basic-Site/Creating-Your-First-Template-and-Content-Node/index.md (hunk 2 above)
Comment (Migaroez) [reply Tutorials/Creating-Basic-Site/Creating-Your-First-Template-and-Content-Node/index.md]: All concepts should be capatilized then?
Document Type
Content Node
Template
... ?

File: Tutorials/Creating-Basic-Site/Creating-Your-First-Template-and-Content-Node/index.md (hunk 1 above)
Comment (sofietoft) [reply Tutorials/Creating-Basic-Site/Creating-Your-First-Template-and-Content-Node/index.md]: Ah, that's a very good point!! 🤔 Didn't think about that..
Let's keep it the way you made it 😁

File: Tutorials/Creating-Basic-Site/Creating-Your-First-Template-and-Content-Node/index.md (hunk 2 above)
Comment (sofietoft) [reply Tutorials/Creating-Basic-Site/Creating-Your-First-Template-and-Content-Node/index.md]: Ah, good question!
I'd say both Document Type and Template should be capitalized.
Let's leave content node as is for now, I think!

**Comment by emmaburstow** [comment]
Hey @Migaroez 

Thanks for thinking to do this - great work and good to see the discussion back and forth too. When you've made the changes as suggested by @sofietoft let me know and I'll hit approve.

Emma

**Comment by sofietoft** [comment]
@Migaroez - Did you have a chance to go through the PR here and capitalize `Document Types` ? 😁

**Comment by sofietoft** [comment]
Thanks @Migaroez 😄 🎉 
Looks great!

**Pull request merged**